packages/python_host/src/one_sdk/app.py

```diff
@@ -80,9 +84,10 @@ def __init__(
 		sf_host_link(self._linker, self)
 
 		wasi = WasiConfig()
-		wasi.inherit_env()
 		wasi.inherit_stdout()
 		wasi.inherit_stderr()
+		wasi.env = [['ONESDK_DEFAULT_USERAGENT', WasiApp.user_agent() ]]
```

Comment:
issue: this now drops the other env variables that were being inherited. We can either inject useragent var into `os.environ` or reconstruct all other env vars of this python process here.

Reply:
Damn, I forgot about this. Thanks

Reply:
I did reconstructing. It's more work for cpu, but feels cleaner.